### PR TITLE
Add file links to model validation doc

### PR DESCRIPTION
**Pull request by mingyu89**
Tested by clicking the links and open the related files.

### CODE REVIEW

**Review by vladimirk-db — APPROVED** [review]

File: {{cookiecutter.project_name}}/databricks-config/README.md [hunk 1]
```diff
@@ -92,12 +92,12 @@ The model validation stack focuses on building a plug-and-play stack component f
 in staging and prod.
 Its central purpose is to evaluate a registered model and validate its quality before deploying the model to Production/Staging.
 
-The model validation job is implemented in `notebooks/ModelValidation`. The model validation stack is defined in 
-`staging/training-job.tf` and `prod/training-job.tf`.
+The model validation job is implemented in `notebooks/ModelValidation](../notebooks/ModelValidation.py). The model validation stack is defined in
```
Comment (arpitjasa-db) [inline {{cookiecutter.project_name}}/databricks-config/README.md:95]:
```suggestion
The model validation job is implemented in notebooks/ModelValidation](../notebooks/ModelValidation.py). The model validation stack is defined in 
```

**Review by arpitjasa-db — APPROVED** [review]
Minor fix, but LGTM! Thanks @mingyu89

File: {{cookiecutter.project_name}}/databricks-config/README.md (hunk 1 above)
Comment (mingyu89) [reply {{cookiecutter.project_name}}/databricks-config/README.md]: WOW great catch! It wasn't a valid link format and did not get  highlighted by intelliJ.

Comment (arpitjasa-db) [reply {{cookiecutter.project_name}}/databricks-config/README.md]: Haha glad my annoying nit-picking tendencies came in handy this time

**Pull request merged**